packages/tldraw/src/lib/ui/context/actions.tsx

```diff
@@ -944,13 +944,22 @@ export function ActionsProvider({ overrides, children }: ActionsProviderProps) {
 				label: 'action.paste',
 				kbd: '$v',
 				onSelect(source) {
-					navigator.clipboard?.read().then((clipboardItems) => {
-						paste(
-							clipboardItems,
-							source,
-							source === 'context-menu' ? editor.inputs.currentPagePoint : undefined
-						)
-					})
+					navigator.clipboard
+						?.read()
+						.then((clipboardItems) => {
+							paste(
+								clipboardItems,
+								source,
+								source === 'context-menu' ? editor.inputs.currentPagePoint : undefined
+							)
+						})
+						.catch(() => {
+							addToast({
```

Comment:
Looks like `read` only throws [one error type](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/read#exceptions), so even though we catch everything it should only be the `NotAllowed` error.